src/azul/logging.py

```diff
@@ -1,14 +1,52 @@
 import logging
 
+from more_itertools import (
+    one,
+)
+
 import azul
 from azul.chalice import (
     AzulChaliceApp,
 )
 
 
+class LambdaLoggerFilter(logging.Filter):
+
+    def __init__(self, app: AzulChaliceApp = None):
+        super().__init__()
+        # We refer to the mutable app instance rather than the lambda_context
+        # because the lambda_context is initilized with `None`. We want to use
+        # the updated lambda_context value, not the singleton `None`.
+        self.app = app
+
+    def filter(self, record):
+        try:
+            record.aws_request_id = self.app.lambda_context.aws_request_id
+        except AttributeError:
```

Comment:
Don't like that this relies on None.lambda_context raising attribute error. Test for self.app is None separately from testing for aws_request_id being absent.